docs/gsoc_report.md

```diff
@@ -0,0 +1,132 @@
+# Google Summer of Code 2021 report
+
+The goal of the Wagtail Live project was to propose a flexible open-source liveblogging tool.
+
+A live blog is a blog providing real-time updates on an ongoing event.
+
+As part of the GSOC program, my mentors and I have spent summer 2021 working on it.
```

Comment:
Although it's called "Google Summer of Code", I think it's best practice to specify the months, rather than the season, as it's winter for half the planet :)